openstudiocore/src/energyplus/ReverseTranslator.cpp

```diff
@@ -791,6 +791,11 @@ boost::optional<ModelObject> ReverseTranslator::translateAndMapWorkspaceObject(c
       modelObject = translateScheduleDayHourly(workspaceObject);
       break;
     }
+  case openstudio::IddObjectType::Schedule_File:
```

Comment:
I guess we should have a no-op translation for the ExternalFile.  Possibly that could be implemented some day to do some file copying magic?